config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -227,6 +245,51 @@ periodics:
     testgrid-tab-name: gce-windows-containerd-master
     description: Runs tests on a Kubernetes cluster with Windows containerd nodes on GCE
 
+- name: ci-kubernetes-e2e-windows-node-throughput
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-gce-windows: "true"
+    preset-load-gce-windows: "true"
+  annotations:
+    testgrid-dashboards: google-windows
+    testgrid-tab-name: windows-gce-node-throughput
+  decorate: true
+  decoration_config:
+    timeout: 1h
+  extra_refs:
+  - org: YangLu1031
+    repo: gce-k8s-windows-testing
```

Comment:
Instead of introducing yet another repo, can you send a PR with the changes you need to yujuhong/gce-k8s-windows-testing?

We should migrate that stuff to a more official repo soon...

Reply:
i created an org gcpwin, and imported the original repo, https://github.com/gcpwin/gce-k8s-windows-testing. you are all invited to join the org.

Reply:
Yang, please send a PR with your changes to that new gcpwin repository for now.

I created https://github.com/kubernetes-sigs/windows-testing/pull/120 to put this code in a centralized sig-windows repo but it's not ready quite yet. Once that's in we can also integrate your PR, then update all of the test jobs to refer to it.

Reply:
created [PR for gcpwin](https://github.com/gcpwin/gce-k8s-windows-testing/pull/1)

Reply:
PR merged. We can start with this, I'll follow up with a PR that switches to using https://github.com/kubernetes-sigs/windows-testing instead.